hazelcast/src/main/java/com/hazelcast/spi/properties/ClusterProperty.java

```diff
@@ -1626,6 +1629,140 @@ private int getWhenNoSSLDetected() {
     public static final HazelcastProperty MOBY_NAMING_ENABLED
             = new HazelcastProperty("hazelcast.member.naming.moby.enabled", true);
 
+    /*
```

Comment:
This needs to be updated too: https://github.com/hazelcast/hazelcast/blob/26e776194b3f6db57aed59510ecd397158585e5c/site/docs/operations/configuration.md?plain=1#L186

Reply:
Probably no need to: https://github.com/hazelcast/hazelcast/pull/19036